recipes/qhull/all/conandata.yml

```
@@ -1,27 +1,27 @@
 sources:
+  "8.1-alpha4":
+    url: "https://github.com/qhull/qhull/archive/refs/tags/v8.1-alpha4.tar.gz"
```

```suggestion
  "8.1.alpha4":
    url: "https://github.com/qhull/qhull/archive/refs/tags/v8.1.alpha4.tar.gz"
```

https://github.com/qhull/qhull/tags

There is a duplication over alpha4 tag. The latest uses a dot, not a hyphen.

Usually alpha/beta are not attractive to be added in CCI, but it's clear the alpha state is pretty long, 4 years since last stable release, so it's fine.